curl/src/utils/message.generator.ts

```diff
@@ -0,0 +1,9 @@
+import {Injectable} from '@nestjs/common';
+
+@Injectable()
+export class MessageGenerator {
+    static generate(sizeInBytes: number) {
+        // Generate the string by repeating the character 'a'
+        return 'a'.repeat(sizeInBytes);    }
```

Comment:
can be nicer than 'aaaaaaaaaaaaaa' but not required
message = ''.join(random.choices(string.ascii_uppercase +
                             string.digits, k=N))

Reply:
Leaving it like this because there is no simple lib in nest.js that does this.